guides/zkml.md

```
@@ -12,13 +12,13 @@
 
 ## Prerequisites
 
-- An operational [EVM](/tutorials/beaconkit.md) using Rollkit (see next step)
+- An operational EVM using Rollkit (see next step)
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: ExecutionCallout

(MD033, no-inline-html)